templates/lookup/ifupdown__dynamic_packages.j2

```diff
@@ -16,6 +16,4 @@
 {%     endif %}
 {%   endfor %}
 {% endif %}
-{% if ifupdown__tpl_packages %}
 {{ ifupdown__tpl_packages | unique | to_yaml }}
-{% endif %}
```

Comment:
Just tested and it works OK. Werid, but fine, I guess. Not sure why it errorer before.